src/psyclone/psyir/transformations/omp_loop_trans.py

```diff
@@ -131,6 +138,59 @@ def __init__(self, omp_directive="do", omp_schedule="auto"):
     def __str__(self):
         return "Adds an OpenMP directive to parallelise the target loop"
 
+    def _add_asynchronicity(self, node: Loop, instance: Directive):
+        '''
+        TODO
```

Comment:
:-)

Reply:
Added a docstring - if you feel it needs more detail let me know.